activestorage/lib/active_storage.rb

```diff
@@ -46,6 +46,8 @@ module ActiveStorage
   mattr_accessor :verifier
   mattr_accessor :variant_processor, default: :mini_magick
 
+  mattr_accessor :variant_transformer
```

Comment:
Note to self: we should probably convert all these `mattr_accessor` to `attr_accessor`.